integration-tests/path-prefix/cypress/integration/path-prefix.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+
+describe(`Production pathPrefix`, () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it(`returns 200 on base route`, () => {
+    cy.url().should('include', '/blog')
+  })
+
+  describe('navigation', () => {
+    let anchor
+    beforeEach(() => {
+      anchor = cy.get('a[data-test-id="page-2-link"]')
```

Comment:
does this work? because it was quite big hurdle for me when I started to update cypress tests that cypress command aren't really sync - for example check https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html particularly this big warning block:
```
 Return Values
You cannot assign or work with the return values of any Cypress command. Commands are enqueued and run asynchronously.
```

Reply:
Ha, it did, but that seems like non-idiomatic Cypress. Will address!

Reply:
especially stuff in this section https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Aliases would say it's not cypress way (unfortunately)

Reply:
Fixed this :) Really good find, and thanks for the link!